src/ensembl/src/content/app/browser/browser-nav/BrowserNavBarControls.test.tsx

```diff
@@ -18,33 +18,19 @@ describe('BrowserNavBarControls', () => {
       mount(
         <BrowserNavBarControls
           browserNavStates={browserNavStates}
-          disabled={faker.random.boolean()}
+          shouldBeOpaque={faker.random.boolean()}
```

Comment:
`shouldBeOpaque` is a very specific name describing how the disabled status of this particular component is currently shown. Why did you prefer it over `disabled`? I think `disabled` was far better.

Reply:
Because it is actually not disabled unlike how it was before. You can click and interact with it just like a normal input.

Reply:
> You can click and interact with it just like a normal input as there is no overlay on it.

I am confused. I went and asked Andrea, and she says the buttons at this point are effectively disabled. A click on an active button (which is now "knocked-back blue") will not execute what this button does, but will just remove the semitransparent screen that is overlaying the buttons.

Reply:
I just talked with Andrea about this. The region editor and the region field are clickable when either of them are active. However, the navigation icons should have an overlay on top of them.

Reply:
There has been a change again. The region will be clickable when the region editor is active. However, the region editor or the navigation icons won't be clickable when the region field is active. The user will need to click away from the region field in order to use them i.e. need an extra click.

Reply:
I am a bit confused about the description. Am I correct in understanding that this is what will be happening?

![Untitled Diagram](https://user-images.githubusercontent.com/6834224/70323283-4f818900-1824-11ea-9c23-9e1ee98463da.png)

Reply:
Yes, that's correct.

Reply:
✅